python/cudf_polars/cudf_polars/streaming/parallel.py

```
@@ -122,7 +125,7 @@ def evaluate_streaming(
     # Clear source info cache in case data was overwritten
     _clear_source_info_cache()
 
-    from cudf_polars.experimental.rapidsmpf.core import evaluate_logical_plan
+    from cudf_polars.streaming.actor_graph.core import evaluate_logical_plan
```

Ruff (0.15.13)

[warning] `import` should be at the top-level of a file

(PLC0415)